src/styles/globals.scss

```diff
@@ -1,27 +1,18 @@
-@use "sass:map";
-@import "./generics/fonts";
-@import "./components/home";
-@import "./components/header";
-@import "./elements/ul";
-@import "./components/componentCard";
+@use 'sass:map';
+@import './generics/fonts';
+@import './components/home';
+@import './components/header';
+@import './elements/ul';
+@import './components/component-card';
+@import './components/component-grid';
```

Comment:
Hopefully, once we get Prettier figured out, these single quotes will stay double for you!!